six/six/api.cpp

```diff
@@ -68,8 +81,16 @@ create_t *loadAndCreate(const char *dll, const char *python_home, char **error)
     }
     return create;
 }
+
 six_t *make2(const char *python_home, char **error)
 {
+
+    if (six_backend != NULL) {
+        std::string err_msg = "Six already initialized!";
+        *error = strdup(err_msg.c_str());
```

Comment:
it's an error case, so it's not a big deal (and not likely to be hit), but this is inefficient.  Creates a `std::string` on the stack, allocates memory to store "six already...", and then just re-allocates the memory for return.

*error = strdup("Six already initialized") would be fine.